core/src/main/java/org/apache/carbondata/core/scan/result/vector/CarbonColumnVector.java

```diff
@@ -114,4 +114,14 @@
 
   void setLazyPage(LazyPageLoader lazyPage);
 
+  CarbonColumnVector getColumnVector();
+
+  CarbonColumnVector getChildrenVector();
+
+  void putArrayObject();
+
+  int getIndex();
```

Comment:
please add comments for interfaces, so the new person who implements will know the usage

Reply:
done

Reply:
done